.pre-commit-config.yaml

```diff
@@ -69,6 +69,8 @@ repos:
     hooks:
       - id: prettier
         types: [yaml]
+        endOfLine: auto
+        proseWrap: "always"
```

Comment:
Prettier config goes into a `.prettierrc` file. For an example, see the CalHR repo

Reply:
Thank you, will take a look. The docs and instructions were fairly vague on where this should go.

Reply:
Here is documentation on how to configure prettier: https://prettier.io/docs/en/configuration

Reply:
Thanks for the link, I looked at that earlier. It wasn't clear to me that I should be creating a new configuration file to add these to versus adding these arguments to existing files. I searched our repo for `# .prettierrc or .prettierrc.yaml` and couldn't find it so I thought we were not in need of the file. Good to know I should have just created it. Thanks again!

Reply:
Yeah, we need to create it. It would be technically possible to pass it in via CLI args in the pre-commit config, but I think that's not the best way to do it